pkg/trie/triedb/codec/node.go

```diff
@@ -109,7 +109,7 @@ type (
 	// Branch could has or not has values
 	Branch struct {
 		PartialKey []byte
-		Children   [16]MerkleValue
+		Children   [ChildrenCapacity]MerkleValue
```

Comment:
Does this mean that we can easily make this support a binary trie, aka 2 children?

Reply:
It would be better if go support [const generics](https://practice.course.rs/generics-traits/const-generics.html) as rust, but yeah, if we change that constant we would support any trie child amount